npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/pnpm_lockfile_updater.rb

```diff
@@ -43,6 +43,7 @@ def updated_pnpm_lock_content(pnpm_lock)
         FORBIDDEN_PACKAGE = /ERR_PNPM_FETCH_403[ [^:print:]]+GET (?<dependency_url>.*): Forbidden - 403/
         MISSING_PACKAGE = /ERR_PNPM_FETCH_404[ [^:print:]]+GET (?<dependency_url>.*): Not Found - 404/
```

Comment:
Since the error output differs for just `Not Found`, can we optimize the `MISSING_PACKAGE` itself. Something like below:

```bash
  MISSING_PACKAGE = /ERR_PNPM_FETCH_404[ [^:print:]]+GET (?<dependency_url>.*):(?: Not Found)? - 404/
```

Reply:
@honeyankit , i have fixed the string reg ex, thanks for the suggestion.